apple/internal/partials/swift_dylibs.bzl

```diff
@@ -171,11 +181,25 @@ def _swift_dylibs_partial_impl(
                 platform_name,
                 binaries_to_check,
                 output_dir,
+                strip_bitcode,
             )
 
             bundle_files.append((processor.location.framework, None, depset([output_dir])))
 
-            swift_support_file = (platform_name, output_dir)
+            swift_support_output_dir = intermediates.directory(
+                ctx.actions,
+                ctx.label.name,
+                "swiftlibs_for_swiftsupport",
+            )
+            # We're not allowed to modify stdlibs that are used for Swift
+            # Support --- ignoring the bitcode mode here.
+            _swift_dylib_action(
```

Comment:
what is the perf hit on running this multiple times? probably nbd since this should only be for "release" builds, but I'm curious if it's a problem

Reply:
Didn't see any difference with our app. I'd guess each set of stdlibs would be cached twice and that's it.

And you're right — bundling Swift Support is only needed for App Store builds anyway.